pylot/utils.py

```diff
@@ -11,6 +11,23 @@
 FRONT_SEGMENTED_CAMERA_NAME = 'front_semantic_camera'
 TOP_DOWN_SEGMENTED_CAMERA_NAME = 'top_down_semantic_camera'
 
+# Semantic Labels
+LABEL_2_PIXEL = {
```

Comment:
Please move the map to https://github.com/erdos-project/pylot/blob/master/pylot/perception/segmentation/utils.py#L5